src/reducers/index.ts

```diff
@@ -1,3 +1,16 @@
-const rootReducer = (state, _) => state
+const initialState = {
+    token: '',
+}
+
+const rootReducer = (state = initialState, action) => {
```

Comment:
also, I'm not expecting a rootReducer to manage specific action, this should be handled by other lower reducer. A rootReducer should just combine those many reducers, cf https://github.com/HugoGresse/hacs-repositories/blob/master/src/rootReducer.ts

Reply:
Yes, that was planned, but since for the moment the only info used with the store is the token, I didn't take the occasion yet to split in multiple reducers